src/Core/Domain/Product/Command/UpdateProductStatusCommand.php

```diff
@@ -46,15 +47,23 @@ class UpdateProductStatusCommand
     private $enable;
 
     /**
-     * UpdateProductStatusCommand constructor.
-     *
+     * @var ShopConstraint
+     */
+    private $shopConstraint;
+
+    /**
      * @param int $productId
      * @param bool $enable
+     * @param ShopConstraint $shopConstraint
      */
-    public function __construct(int $productId, bool $enable)
-    {
+    public function __construct(
+        int $productId,
+        bool $enable,
+        ShopConstraint $shopConstraint
```

Comment:
Please define the BC Break

Reply:
Nope no BC for all the product domain code which is experimental

Reply:
actually this status command was introduced earlier (as i see in 2019) so I suppose it should have the BC break indeed?